internal/install/types/recipe.go

```diff
@@ -284,11 +285,20 @@ func toBoolByFieldName(fieldName string, data map[string]interface{}) bool {
 }
 
 func toStringByFieldName(fieldName string, data map[string]interface{}) string {
-	if v, ok := data[fieldName]; ok {
-		return v.(string)
+	out := ""
+	if in, ok := data[fieldName]; ok {
+		switch v := in.(type) {
+		case int:
+			return strconv.Itoa(v)
+		case string:
+			return v
```

Comment:
Noob Go question - does `return v.(string)` make sense here or is the type assertion not as valuable because you're already switching on type?

Reply:
The first part of the switch statement `v := in.(type)` makes `v` a string so the type assertion is not necessary in that block. This is kind of unique to [`switch` by type](https://tour.golang.org/methods/16) in Go.

Reply:
Ooooh that's nice